modin/conftest.py

```diff
@@ -342,6 +348,133 @@ def make_csv_file():
     teardown_test_files(filenames)
 
 
+def create_and_call(filenames, filename, force, nrows, ncols, func: str, func_kw=None):
+    if force or not os.path.exists(filename):
+        df = pandas.DataFrame({f"col{x + 1}": np.arange(nrows) for x in range(ncols)})
+        getattr(df, func)(filename, **func_kw if func_kw else {})
+        filenames.append(filename)
+
+
+def make_default_file(file_type: str):
```

Comment:
I think it is possible to reduce code duplication of this function. Also this function can be moved to the `utils` module.

Reply:
Reduced and moved.